source/tools/kubernetes/dask-helm-chart.md

```diff
@@ -1,2 +1,139 @@
+compute the hash for `<your-password>` and update:
+
+Key | Value 
+---|---
+jupyter.password | `hash(<your-password>)`
+
+<details>
+<summary>Detailed explainations</summary>
+
+`*.images.*` is updated with the RAPIDS "runtime" image from the stable release,
+which includes environment necessary to launch run accelerated libraries in RAPIDS,
+and scaling up and down via dask.
+Note that all scheduler,
+woker and jupyter pods are required to use the same image.
+This ensures that dask scheduler and worker versions match.
+
+`*.env` is required as of release 22.08 as a workaround for limitations in the image.
+May be removed in the future.
+
+`worker.resources` exlicitly requests GPU for each worker pod,
+required by many accelerated libraries in RAPIDS.
+
+`worker.dask_worker` is the launch command for dask worker inside worker pod.
+To leverage the GPU resource each pod has,
+[`dask_cuda_worker`](https://docs.rapids.ai/api/dask-cuda/stable/index.html) is launched in place of the regular `dask_worker`.
+
+</details>
+
+Deploy the modified helm-chart via for single user use:
+```
+helm install rapids-release dask/
+```
+
+This will deploy the cluster with the same topography as dask helm chart,
+see [dask helm chart documentation for detail](https://github.com/dask/helm-chart/blob/main/dask/.frigate).
+
+Note that no `Ingress` is created.
+Custom `Ingress` may be configured to consume external traffic and redirect to corresponding services.
+For simplicify, this guide will setup access to jupyter notebook via port forwarding.
+
+## Running Rapids Notebook
+
+First, setup port forwarding from the cluster to external port:
+
+```bash
+# For Jupyter notebook
+kubectl port-forward --address 127.0.0.1 service/rapids-release-dask-jupyter 8888:80 &
+# For Dask scheduler
+kubectl port-forward --address 127.0.0.1 service/rapids-release-dask-scheduler 8889:80 &
+```
+
+For users accessing the notebooks from remote machine,
+ssh-tunneling is required.
+Otherwise,
+open a browser and access `localhost:8888` for jupyter notebook,
+and `localhost:8889` for dask dashboard.
+Enter password (default is `dask`) and access the notebook environment.
+
+### Notebooks and Cluster Scaling
+
+RAPIDS is a GPU accelerated,
+scalable environment suitable for various data science workflow.
+[`10 Minutes to cuDF and Dask-cuDF`](https://docs.rapids.ai/api/cudf/stable/user_guide/10min.html)
+notebook demonstrates using [`cuDF`'s](https://docs.rapids.ai/api/cudf/stable/) pandas-like API
+to accelerate the workflow on GPU.
+The notebook is hosted in `cudf` folder,
+so be sure to take the time and walk through the notebook!
+
+<!-- TODO: Image to demonstrate the dashboard with the usage of the workers -->
+
+In cluster server,
+execute the following to retrieve the IP address of the scheduler:
+```
+kubectl get svc -l component=scheduler,release=rapids-release
+```
+
+```python
+from dask.distributed import Client
+
+client = Client(<your-scheduler-ip-address:8786>)
```

Comment:
The helm chart is clever here and preconfigures this address so you don't need to set it. We should remove the address and tell users about this because it is cool!

```suggestion
client = Client()
```